doc/user-guide/04-scenarios.adoc

```diff
@@ -67,6 +67,106 @@ a single confidential command to let the redirection also apply for 'set -x'.
+
+There are two options how to boot the rescue environment on IBM Z (s390x) under
+z/VM created using the `OUTPUT=RAMDISK` setting.
+
+1. Using the `kexec` tool as described in link:../../usr/share/rear/conf/templates/RESULT_usage_RAMDISK.txt[RESULT_usage_RAMDISK.txt],
+   if you have access to a running Linux instance.
+
+2. Using the `zipl` tool for management of boot devices on s390/s390x.
+
+=== Booting ReaR rescue initramfs on IBM Z (s390x) under z/VM with `zipl`
+
+After creating the rescue image with `rear mkrescue`, move the kernel and
+initramfs to a directory (denoted by `REAR_DIRECTORY` in this example) on the
+target device to boot from.
+
+NOTE: See `zipl(8)` for all device types supported by the `--target` option.
+      If you want to use a tape device, see `zipl(8)` to use the `--tape`
+      option instead. Note that the `--tape` option with ReaR is untested so
+      contributions to this guide are welcome! Also, it is not possible to use
+      `zipl` with the z/VM reader device.
+
+IMPORTANT: The following method will overwrite the boot configuration on a device
+           which contains the given `--target` directory!
+
+Use the following command to make the device with `REAR_DIRECTORY` bootable
+with `zipl`:
+```
+# zipl --target=BOOTMAP_INSTALL_DIRECTORY \
+       --parameters='root=/dev/ram0 ro' \
+       --image=REAR_DIRECTORY/kernel-HOSTNAME \
+       --ramdisk=REAR_DIRECTORY/initramfs-HOSTNAME.img
+Run /lib/s390-tools//zipl_helper.device-mapper BOOTMAP_INSTALL_DIRECTORY
+Building bootmap in 'BOOTMAP_INSTALL_DIRECTORY'
+Adding IPL section
+  initial ramdisk...: REAR_DIRECTORY/initramfs-HOSTNAME.img
+  kernel image......: REAR_DIRECTORY/kernel-HOSTNAME
+  kernel parmline...: 'root=/dev/ram0'
+  component address:
+    internal loader.: 0x0000a000-0x0000dfff
+    parameters......: 0x00009000-0x00009fff
+    kernel image....: 0x00010000-0x00cdbfff
+    parmline........: 0x00cdc000-0x00cdcfff
+    initial ramdisk.: 0x00cf0000-0x10980fff
+    environment blk.: 0x00cec000-0x00cecfff
+Preparing boot device for CCW- and LD-IPL: dasda.
+Done.
+```
+You must set `BOOTMAP_INSTALL_DIRECTORY` to any directory on the same device
+which also contains `REAR_DIRECTORY` to store the boot data.  The machine will
+not boot if `BOOTMAP_INSTALL_DIRECTORY` and `REAR_DIRECTORY` are on different
+devices!
+
+The bootloader in the example was installed on the `/dev/dasda` disk device
+because it contains the `BOOTMAP_INSTALL_DIRECTORY` directory.
+
+If you want to boot the device with ReaR recovery from a running Linux
+instance, you can use the `chreipl` tool to select a new default boot device
+and reboot:
+```
+# chreipl REAR_DIRECTORY
+Re-IPL type: ccw
+Device:      0.0.0120
+Loadparm:    ""
+Bootparms:   ""
+clear:       0
+# reboot
+...
+```
+
+If you do not have a running Linux instance at hand, you can force the boot
+from the given device using the z/VM Control Program (CP) directly, e.g. using
+a serial console.
```

Comment:
I don't think there is a serial console on the S/390 - we are not in Unix world like on the PowerPC/PReP or Sparc machines. There is 3270 which AFAIK is a very different beast. There is also 3215 which may be sufficiently similar to a serial connection but I don't think it is a serial connection really, not sure.

Reply:
Thank you!  Changed in https://github.com/rear/rear/commit/873884f3831bc1d4345ce247df2e3196bd15b1d6.